upgrade-1.x-to-2.x.sh

```diff
@@ -988,9 +988,9 @@ rm -rf /mnt/data/resin-data/resin-supervisor
 sync
 if [ "$NOREBOOT" == "no" ]; then
     # Reboot into new OS
-    log "Rebooting into new OS in 5 seconds..."
+    log "Rebooting into new OS..."
     progress 100 "Update successful, rebooting"
```

Comment:
Was the sleep intended to ensure the progress message makes it out before the reboot? If so, it may be worth prepending the reboot with the same `sleep 5;`

Reply:
@xginn8 done with a `systemd-run` native way.